src/stores/QtumPredictionStore.js

```diff
@@ -0,0 +1,72 @@
+import { observable, action, runInAction, reaction } from 'mobx';
+import _ from 'lodash';
+import { Token, OracleStatus, AppLocation } from '../constants';
+import { queryAllOracles } from '../network/graphQuery';
+import Oracle from './models/Oracle';
+
+
+export default class AllPredictionsStore {
+  @observable loading = true
+  @observable loadingMore = false
+  @observable list = []
+  @observable hasMore = true
+  @observable skip = 0
+  limit = 50
+
+  constructor(app) {
+    this.app = app;
+    reaction(
+      () => this.app.sortBy, // when 'sortBy' changes
```

Comment:
suggestion: why don't we have separate `sortBy` per each page? that way if the user navigates out and back in to the page it will stay on whichever one they chose previously.

Reply:
move this to #718, want to land this first.